libmozevent/utils.py

```diff
@@ -175,16 +184,31 @@ class AsyncRedis(object):
     @staticmethod
     async def connect():
         if AsyncRedis.redis.get(None) is None:
+            retry_on_error = [
+                ConnectionError,
+                ExecAbortError,
+                PubSubError,
+                RedisError,
+                ResponseError,
+                TimeoutError,
+                WatchError,
+            ]
```

Comment:
is there a way to say "retry on everything" without listing the specific exceptions?